unittest.cfg

```diff
@@ -1,13 +1,19 @@
 [unittest]
 verbose = 2
-plugins = flufl.testing.nose
+plugins =
+    nose2.plugins.prof
 
 [log-capture]
 always-on = False
+clear-handlers = True
+date-format = None
+; format = %%(name)s: %%(levelname)s: %%(message)s
+format = %%(asctime)s: %%(name)s: %%(levelname)s: %%(message)s
+filter = -nose
+log-level = DEBUG
 
-[flufl.testing]
-always-on = True
-package = aiosmtpd
-setup = aiosmtpd.testing.helpers.setup
-teardown = aiosmtpd.testing.helpers.teardown
-start_run = aiosmtpd.testing.helpers.start
+[profiler]
+always-on = False
+filename = _dynamic/cPROFILE
```

Comment:
Combined with the renaming Python script in tox.ini, this allows per-testenv cProfile data.

Also shoving them into `_dynamic` so can easily be cleaned up by removing that single directory.